worker-message-prioritization-distribution-container/pom.xml

```diff
@@ -0,0 +1,233 @@
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+
+                    <!-- Start the containers in pre-integration-test phase -->
+                    <execution>
+                        <id>start-docker-containers</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>start</goal>
+                        </goals>
+                    </execution>
+
+                    <!-- Stop the containers in post-integration-test phase -->
+                    <execution>
+                        <id>stop-docker-containers</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>stop</goal>
+                            <goal>volume-remove</goal>
+                        </goals>
+                    </execution>
+
+                    <!-- Push the docker image to the registry in the deploy phase -->
+                    <execution>
+                        <id>docker-image-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                    </execution>
+
+                </executions>
+
+                <configuration>
+                    <autoPull>on</autoPull>
+                    <containerNamePattern>%a-%t</containerNamePattern>
+                    <verbose>true</verbose>
+                    <images>
+
+                        <!-- Run the RabbitMQ image -->
+                        <image>
+                            <alias>rabbitmq</alias>
+                            <name>${dockerHubPublic}/library/rabbitmq:3-management</name>
+                            <run>
+                                <ports>
+                                    <port>${rabbitmq.ctrl.port}:15672</port>
+                                    <port>${rabbitmq.node.port}:5672</port>
+                                </ports>
+                                <wait>
+                                    <log>Server startup complete</log>
+                                    <http>
+                                        <url>http://${docker.host.address}:${rabbitmq.ctrl.port}</url>
+                                    </http>
+                                    <time>120000</time>
+                                    <shutdown>500</shutdown>
+                                </wait>
+                                <log>
+                                    <enabled>true</enabled>
+                                </log>
+                            </run>
+                        </image>
+
+                        <image>
+                            <name>${dockerWorkerFrameworkOrg}worker-message-prioritization-distribution${dockerProjectVersion}</name>
```

Comment:
I don't see any workerframework org currently on dockerhub so we might need to create one:

https://hub.docker.com/search?q=workerframework

Reply:
https://hub.docker.com/orgs/workerframework